app/api/events.py

```diff
@@ -523,7 +511,7 @@ def get_id(view_kwargs):
             view_kwargs['id'] = None
 
     if view_kwargs.get('tax_id') is not None:
-        tax = safe_query(db, Tax, 'id', view_kwargs['tax_id'], 'tax_id')
+        tax = safe_query(Tax, 'id', view_kwargs['tax_id'], 'tax_id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)